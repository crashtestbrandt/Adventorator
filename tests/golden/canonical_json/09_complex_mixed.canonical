{"metadata":{"version":1},"users":[{"active":true,"name":"Alice"},{"active":false,"name":"Bob"}]}