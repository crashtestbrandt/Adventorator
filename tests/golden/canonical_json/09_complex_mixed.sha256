e42fe2fef538ca533de0a528aef614668810ab82314007ba27792ed1e49f2426
