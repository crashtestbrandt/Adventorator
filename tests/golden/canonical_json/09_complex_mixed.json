{
  "users": [
    {
      "name": "Alice",
      "active": true
    },
    {
      "name": "Bob",
      "active": false
    }
  ],
  "metadata": {
    "version": 1,
    "created": null
  }
}